src/annotator/anchoring/test/text-range-test.js

```diff
@@ -325,5 +347,19 @@ describe('annotator/anchoring/text-range', () => {
         }, 'Point is not in an element or text node');
       });
     });
+
+    describe('fromOffsets', () => {
```

Comment:
You've been remarkably consistent in your test naming. Do you think there's any value in having a convention for naming tests for static methods? I don't know what that convention would _be_.

Reply:
I treat static methods just like regular functions, since that's what they are. Only methods (where `this` matters) have a `#prefix`. Incidentally, that convention might become a bit confusing in future because `#` is starting to be used as a sigil for private fields/methods in classes in the JS language. Sigh.